src/3b_count_aca.jl

```diff
@@ -107,32 +97,82 @@ end
             reef_features[target_row, :region] = reg
         end
     end
-    
+
     target_flats = nothing
     target_slopes = nothing
     GC.gc()
 end
 
+# Calculate the proportion of each reef area that meets 0.95 suitability criteria
 valid_locs = reef_features.Area_HA .!= 0
 reef_features[valid_locs, :flat_scr] .= round.(reef_features[valid_locs, :flat_ha] ./ reef_features[valid_locs, :Area_HA], digits=4)
 reef_features[valid_locs, :slope_scr] .= round.(reef_features[valid_locs, :slope_ha] ./ reef_features[valid_locs, :Area_HA], digits=4)
 
 for reg in REGIONS
-    # Determine float/slope score for each region
+    # `scr` then gives an index to find the reefs within a region that have the most
+    # proportion of their area suitable for deployments
     target_reg = reef_features.region .== reg
     reef_features[target_reg, :flat_scr] .= round.(reef_features[target_reg, :flat_scr] / maximum(reef_features[target_reg, :flat_scr]), digits=4)
     reef_features[target_reg, :slope_scr] .= round.(reef_features[target_reg, :slope_scr] / maximum(reef_features[target_reg, :slope_scr]), digits=4)
 end
 
-# Have to write out results as shapefile because of ArcGIS not handling GeoPackages for
-# some reason...
+# Have to write out results as shapefile because of ArcGIS not handling GeoPackages
 GDF.write(
-    joinpath(QGIS_DIR, "reef_suitability.shp"),
-    reef_features[:, [:geometry, :region, :reef_name, :flat_ha, :slope_ha, :Area_HA, :n_flat, :n_slope, :flat_scr, :slope_scr, :UNIQUE_ID]];
-    layer_name="reef_suitability",
-    geom_columns=(:geometry,),
+    joinpath(ACA_QGIS_DIR, "reef_suitability.gpkg"),
+    reef_features[:, [
+        :geometry,
+        :region,
+        :reef_name,
+        :flat_ha,
+        :slope_ha,
+        :Area_HA,
+        :n_flat,
+        :n_slope,
+        :flat_scr,
+        :slope_scr,
+        :UNIQUE_ID
+    ]];
     crs=EPSG(4326)
 )
 
-subdf = reef_features[:, [:region, :reef_name, :flat_ha, :slope_ha, :Area_HA, :n_flat, :n_slope, :flat_scr, :slope_scr, :UNIQUE_ID]]
-CSV.write("../qgis/potential_reef_areas.csv", subdf)
+subdf = reef_features[:, [
+    :region,
+    :reef_name,
+    :flat_ha,
+    :slope_ha,
+    :Area_HA,
+    :n_flat,
+    :n_slope,
+    :flat_scr,
+    :slope_scr,
+    :UNIQUE_ID
+]]
+CSV.write(joinpath(ACA_QGIS_DIR, "potential_reef_areas.csv"), subdf)
```

Comment:
Consistent with 3a: Comment and express subdf in consistent format.
subdf = reef_features[:, [:region, :reef_name, :flat_ha, :slope_ha, :Area_HA, :n_flat, :n_slope, :flat_scr, :slope_scr, :UNIQUE_ID]]
```suggestion
# Write data to csv file
subdf = reef_features[:, [
    :region,
    :reef_name,
    :flat_ha,
    :slope_ha,
    :Area_HA,
    :n_flat,
    :n_slope,
    :flat_scr,
    :slope_scr,
    :UNIQUE_ID
]]
CSV.write(joinpath(ACA_QGIS_DIR, "potential_reef_areas.csv"), subdf)
```